src/develop.js

```diff
@@ -81,16 +80,17 @@ cmd-shift-o to run the package out of the newly cloned repository.\
     }
 
     installDependencies(packageDirectory, options) {
-        process.chdir(packageDirectory);
-        const installOptions = _.clone(options);
+      process.chdir(packageDirectory);
 
-        return new Install().run(installOptions);
+      return new Install().run({...options});
```

Comment:
Again, almost sure I checked all functions and what they do, and also, I don't think it's all that obscure; specifically, I feel that merely passing a shallow copy without saying anything about why that's necessary is way more obscure than the spread syntax way of creating a shallow copy.

Reply:
I totally agree that doing a shallow copy with no comment originally is more obscure, because when I wrote that even I was wondering why they ever needed to do that.

But I'm sure you checked the functions and what they do because I couldn't find any differing behavior in your changes which is awesome.